### PR TITLE
CallTaker Regression Fixes

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Internationalization and OTP2 have created 2 bugs in the call taker UI.

The first the add intermediate stop being broken. #604
The second was the four time adjustment buttons not working.

Both have been fixed via simple patches

### CODE REVIEW

File: lib/actions/plan.js [hunk 1]
```diff
@@ -96,17 +109,22 @@ export function planNext () {
...
-      date: moment(currentQuery.date).add('days', 1).format(OTP_API_DATE_FORMAT),
+      date: moment(currentQuery.date)
+        .add('days', 1)
+        .format(OTP_API_DATE_FORMAT),
       departArrive: 'ARRIVE',
-      startTransitStopId: getFirstStopId(itinerary),
       time: SERVICE_BREAK
     }
+    if (!state.otp?.config?.api?.v2) {
+      params.startTransitStopId = getFirstStopId(itinerary)
```
Comment (daniel-heppner-ibigroup) [inline lib/actions/plan.js:125]: This if statement is repeated several times, could it just go into the `getFirstStopId` function?

**Review by daniel-heppner-ibigroup — APPROVED** [review]

File: lib/actions/plan.js [hunk 2]
```diff
@@ -28,30 +31,32 @@ function offsetTime (itinerary, unixTime) {
...
     if (isPlanningFirst(currentQuery)) date.subtract('days', 1)
     const params = {
       date: date.format(OTP_API_DATE_FORMAT),
       departArrive: 'DEPART',
-      startTransitStopId: getFirstStopId(itinerary),
       time: SERVICE_BREAK
     }
+    if (!state.otp?.config?.api?.v2) {
+      params.startTransitStopId = getFirstStopId(itinerary)
+    }
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/actions/plan.js:59]: OTP2 doesn't support `startTransitStopId` and if you provide it you get no itineraries!

File: lib/actions/plan.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/actions/plan.js:125]: I tried to make this work but the problem is the field can't be set at all and even undefined creates a value!

**Review by binh-dam-ibigroup — APPROVED** [review]

**Comment by miles-grant-ibigroup** [comment]
Assigning to @demory so you can have a look

**Pull request merged**